### PR TITLE
Sitemap (I forgot that these are what they're called)

### PR DESCRIPTION
**Pull request by MegaApplePi**
- updates and replaces `/wiki/Index`
- sorted alphabetically by directory
- the links use full path names relative to the root directory (so clicking on them in GitHub should lead you to a directory listing on GitHub)

### CODE REVIEW

**Comment by TPGPL** [comment]
Maybe we could add a system to auto-update this article basing on directories. Like we would use it once a month or something

**Comment by sr229** [comment]
@TPGPL the most viable way is coding a GitHub bot to make edits based on repository status, this can be done really quick but our problem would be hosting

File: wiki/Sitemap/en.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
+# Sitemap
+
```
Comment (TPGPL) [inline wiki/Sitemap/en.md]: Add something like "Complete list of osu!wiki articles. Updated every month."

File: wiki/redirect.yaml [hunk 2]
```diff
@@ -1028,6 +1026,9 @@
 
 "skr":                         "Shortcut_Key_Reference"
 
+"main_page": "Sitemap"
+"index": "Sitemap"
+
```
Comment (TPGPL) [inline wiki/redirect.yaml]: all_articles, main_page

**Comment by MegaApplePi** [comment]
Those are spaces. It was done that way to make it a but easier for me to read in raw text.

**Comment by TPGPL** [comment]
Okay

File: wiki/Sitemap/en.md (hunk 1 above)
Comment (sr229) [reply wiki/Sitemap/en.md]: I don't think it would be every month, once the bot is live it would be updated every 15 minutes

Comment (MegaApplePi) [reply wiki/Sitemap/en.md]: sitemap is and should already be self-explanatory... a leading paragraph for this isn't needed imo

File: wiki/Sitemap/en.md [hunk 3]
```diff
@@ -0,0 +1,270 @@
+# Sitemap
```
Comment (TPGPL) [inline wiki/Sitemap/en.md]: Add some tags? :thinking:

**Pull request merged**